diff --git a/crates/uv-dev/Cargo.toml b/crates/uv-dev/Cargo.toml
index 6ca862d45..15beb4fe2 100644
--- a/crates/uv-dev/Cargo.toml
+++ b/crates/uv-dev/Cargo.toml
@@ -81,7 +81,6 @@ name = "uv-dev"
 required-features = ["dev"]

 [features]
-default = ["performance"]
 # Actually build the dev CLI.
 dev = []
 performance = ["performance-memory-allocator"]
diff --git a/crates/uv/Cargo.toml b/crates/uv/Cargo.toml
index e290b9892..8db459af5 100644
--- a/crates/uv/Cargo.toml
+++ b/crates/uv/Cargo.toml
@@ -158,7 +158,7 @@ nix = { workspace = true }
 uv-unix = { workspace = true }

 [features]
-default = ["performance", "test-defaults"]
+default = ["test-defaults"]
 native-auth = []
 # Use better memory allocators, etc.
 performance = ["performance-memory-allocator"]

